### PR TITLE
validation applied

### PR DESCRIPTION
**Pull request by provishalk**
hi

### CODE REVIEW

File: src/controllers/employee.controller.js [hunk 1]
```diff
@@ -1,8 +1,16 @@
 const Employee = require("../models/employee.model")
-module.exports.addEmployee = (req,res) => {
-    if (req.body.FirstName.trim().length === 0
-        || req.body.LastName.trim().length === 0
-        || req.body.Status.trim().length === 0
+module.exports.addEmployee = async (req,res) => {
+    let contact = req.body.Contact.trim()
```
Comment (shivdotpy) [inline src/controllers/employee.controller.js]: In Javascript we use camel-casing:
SO you need to change variable names from:
Contact => contact

File: src/controllers/employee.controller.js [hunk 2]
```diff
@@ -1,8 +1,16 @@
...
+    contact = contact.replace(/ +/g, "");  //Remove white space between numbers
+    const checkIsNumberRagex = /^[0-9]{10}$/; //Ragex Condition
+    
+
+    if (req.body.FirstName.trim().length === 0 //Check Employee Name is empty or not
+        || req.body.LastName.trim().length === 0 //Check Employee Name is empty or not
```
Comment (shivdotpy) [inline src/controllers/employee.controller.js]: LastName => lastName

File: src/controllers/employee.controller.js [hunk 3]
```diff
@@ -1,8 +1,16 @@
...
-    if (req.body.FirstName.trim().length === 0
-        || req.body.LastName.trim().length === 0
-        || req.body.Status.trim().length === 0
+module.exports.addEmployee = async (req,res) => {
+    let contact = req.body.Contact.trim()
+    contact = contact.replace(/ +/g, "");  //Remove white space between numbers
+    const checkIsNumberRagex = /^[0-9]{10}$/; //Ragex Condition
+    
+
+    if (req.body.FirstName.trim().length === 0 //Check Employee Name is empty or not
```
Comment (shivdotpy) [inline src/controllers/employee.controller.js]: FirstName => firstName

File: src/controllers/employee.controller.js [hunk 4]
```diff
@@ -1,8 +1,16 @@
...
+        || req.body.LastName.trim().length === 0 //Check Employee Name is empty or not
+        || req.body.Status.trim().length === 0 //Check Employee status is empty or not
```
Comment (shivdotpy) [inline src/controllers/employee.controller.js]: Status => status

File: src/controllers/employee.controller.js (hunk 1 above)
Comment (provishalk) [reply src/controllers/employee.controller.js]: done

File: src/controllers/employee.controller.js (hunk 2 above)
Comment (provishalk) [reply src/controllers/employee.controller.js]: done

File: src/controllers/employee.controller.js (hunk 3 above)
Comment (provishalk) [reply src/controllers/employee.controller.js]: done

File: src/controllers/employee.controller.js (hunk 4 above)
Comment (provishalk) [reply src/controllers/employee.controller.js]: done

**Review by shivdotpy — APPROVED** [review]

**Pull request merged**